src/plugin/action/join.sjs

```diff
@@ -11,13 +11,25 @@ module.exports = function (client, rawf, emitter) {
                 reject(new Error("No channel given to join action."));
                 return;
             }
+            var info = channel;
+             if (typeof(channel) === 'object'){
```

Comment:
Single quoted string should be double quoted.

Also, strange single space here.